### ISSUE TITLE
Numerical values in the graph view are not displayed properly

### ISSUE THREAD

**Issue by SeanCondon** [onosproject/onos-gui#20]
The values are translated from the
https://github.com/onosproject/onos-config/blob/master/pkg/northbound/admin/admin.proto ChangeValue message type (line 70)

In the gui it is being translated in to a string value by the ChangeValuePipe:
https://github.com/onosproject/onos-gui/blob/master/web/onos-gui/src/app/onos-config/change-value.pipe.ts line 26

FYI - the Pipe is used in 
https://github.com/onosproject/onos-gui/blob/master/web/onos-gui/src/app/onos-config/config-view/container-svg/container-svg.component.html line 34, where **value** is 'piped' to **changevalue**.

Basically there are 3 parts to the change value - as seen in https://github.com/onosproject/onos-config/blob/master/pkg/store/change/typedvalue.go line 69
```
type TypedValue struct {
	Value    []byte
	Type     ValueType
	TypeOpts []int
}
```
Int, Uint, Decimal and Float need to be converted back from this format in to decimal or string values - in TypeScript (preferrably without involving any outside libraries - Angular won't have anything on it - it should be done with ES6 Javascript e.g. http://es6-features.org/ or any other search for ES6 or ECMAScript6)

**Comment by SeanCondon** [comment]
This can now be tested with **ng test** from web/onos-gui folder

**Comment by SeanCondon** [comment]
the function that does the translation has now been transferred in to
https://github.com/onosproject/onos-gui/blob/master/web/onos-gui/src/app/onos-config/change-value.util.ts

and the test cases are in
https://github.com/onosproject/onos-gui/blob/master/web/onos-gui/src/app/onos-config/change-value.util.spec.ts

When you run **ng test --watch=false --browsers=ChromeHeadless** many more unit tests now run (because I have added new ones). Still it is only this one test case failing.

Once this test case has been fixed - **make test** can be added to Travis.

**Comment by SeanCondon** [comment]
Fixed by #27
